tests/test_sklearn_linear_converter.py

```diff
@@ -218,7 +218,7 @@ def test_modified_huber(self):
         np.testing.assert_allclose(model.predict_proba(inputs), hb_model.predict_proba(inputs), rtol=1e-6, atol=1e-6)
 
     @unittest.skipIf(
-        LooseVersion(torch.__version__) < LooseVersion("1.6.0"), reason="Modified Huber loss test requires torch >= 1.6.0"
+        LooseVersion(torch.__version__) < LooseVersion("1.7.0"), reason="Modified Huber loss test requires torch >= 1.7.0"
```

Comment:
Same here.

Reply:
In one of my commit messages, i have something like "turns out, clip is in torch 1.7 not 1.6, so adding back loose versio and putting it to the right number".

Modified Huber Loss is not in 1.6 apparently.

Reply:
Wait but it is working now with torch 1.6, no?

Reply:
No, it's not working with torch 1.6. The test failed, so I looked into it, and that is why I changed it to this.